feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -50,10 +41,11 @@ export class ConfiguratorPriceComponent {
   /**
    * Retrieves formula for quantity with price.
    *
-   * @return {string} - quantity with value price formula
+   * @param {string} formattedQuantity- formatted quantity
+   * @return {string} - price formula
    */
-  get quantityWihPrice(): string {
-    return this.quantity + 'x(' + this.formula?.price?.formattedValue + ')';
+  quantityWihPrice(formattedQuantity: string): string {
```

Comment:
Please correct typo (although it was not introduced by your PR): -> quantityWithPrice

Reply:
Done!